server/src/test/java/org/eclipse/openvsx/eclipse/EclipseServiceTest.java

```diff
@@ -287,6 +301,16 @@ private ResponseEntity<String> mockAgreementResponse() throws IOException {
     
     @TestConfiguration
     static class TestConfig {
+        @Bean
+        public MeterRegistry registry() {
+            return new SimpleMeterRegistry();
+        }
+
+        @Bean
+        public UrlConfigService urlConfigService() {
+            return new UrlConfigService();
+        }
+        
```

Comment:
Can't you add these two beans to the `@MockBean` list? These beans are not used in the test.